### PR TITLE
LITTIL-242 endpoint for receiving feedback

### PR DESCRIPTION
**Pull request by MarcelWildenburg**

### CODE REVIEW

File: src/main/java/org/littil/api/mail/MailService.java [hunk 1]
```diff
@@ -33,6 +37,16 @@ public void sendContactMail(String recipientEmailAddress, String contactMessage,
         send(template);
     }
 
+    public void sendFeedbackMail(FeedbackPostResource feedback, Optional<String> feedbackEmail) {
```
Comment (tubbynl) [inline src/main/java/org/littil/api/mail/MailService.java:40]: i'd prefer function signature consistency with the other public functions here (use String/simple types), don't use Controller DTO's here.

File: src/main/java/org/littil/api/feedback/api/FeedbackResource.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+@Path("/api/v1/feedback")
+@RequestScoped
+@Consumes(MediaType.APPLICATION_JSON)
+@Tag(name = "Feedback")
+public class FeedbackResource {
+
+    @Inject
+    MailService mailService;
+    @Inject
```
Comment (tubbynl) [inline src/main/java/org/littil/api/feedback/api/FeedbackResource.java:27]: should this property be injected into MailService?

File: src/main/java/org/littil/api/mail/MailService.java [hunk 3]
```diff
@@ -33,6 +37,16 @@ public void sendContactMail(String recipientEmailAddress, String contactMessage,
...
+        if (feedbackEmail.isEmpty()) {
+            return;
```
Comment (tubbynl) [inline src/main/java/org/littil/api/mail/MailService.java:42]: but in any case; log.error that no feedback email is set and feedback is ignored (with values)

**Review by tubbynl — CHANGES_REQUESTED** [review]

File: src/main/java/org/littil/api/feedback/api/FeedbackResource.java (hunk 2 above)
Comment (MarcelWildenburg) [reply src/main/java/org/littil/api/feedback/api/FeedbackResource.java]: property moved to MailService

File: src/main/java/org/littil/api/mail/MailService.java (hunk 1 above)
Comment (MarcelWildenburg) [reply src/main/java/org/littil/api/mail/MailService.java]: changed to string types

File: src/main/java/org/littil/api/mail/MailService.java (hunk 3 above)
Comment (MarcelWildenburg) [reply src/main/java/org/littil/api/mail/MailService.java:50]: log warning when email address is not set

**Review by tubbynl — APPROVED** [review]

**Pull request merged**